docs/src/literate/plasticity.jl

```diff
@@ -398,7 +376,7 @@ xlabel!("Maximum deflection [m]")
 
 ## test the result                       #src
 using Test                               #src
-@test norm(u_max[end]) ≈ 0.2544526451    #src
+@test norm(u_max[end]) ≈ 0.255056309     #src
```

Comment:
:)))))))

I guess `symmetric(function_gradient(..))` -> `function_symmetric_gradient(...)` caused this.

Reply:
Shouldn't that literally be the same? https://github.com/Ferrite-FEM/Ferrite.jl/blob/c585e1c29efe2173fd91bfc17d824ae873db1ee9/src/FEValues/common_values.jl#L210:L213

Reply:
Huh, then I don't know why there would be a difference.